packages/ui/src/components/AccountDetails.tsx

```diff
@@ -66,6 +67,75 @@ export const AccountDetails: FC<Props> = ({ defaultName, headerText, onBack, onC
     Object.values(getValues()).filter((val) => val === '').length > 0 ||
     formState.isSubmitting;
 
+  const renderNewAccountForm = () => (
+    <>
+      <Box mt='m'>
+        <Text color='gray.1'
+          variant='b2m'>
+          Account name
+        </Text>
+      </Box>
+      <TextInput inputRef={register({ required: true })}
+        name='accountName'
+        placeholder='Enter account name' />
+      {errors?.accountName &&
+        <Box>
+          <Text color='alert'
+            variant='b3'>
+            {(errors.accountName).type === 'required' && 'Required field'}
+          </Text>
+        </Box>
+      }
+      <Box mt='m'>
+        <Text color='gray.2'
+          variant='b2'>Please enter a new wallet password below to complete account creation.</Text>
+      </Box>
+      <Password label='Password'
+        withConfirm />
+    </>
+  );
+
+  const renderAccountForm = () => (
+    <>
+      <Box mt='m'>
+        <Text color='gray.2'
+          variant='b2m'>
+          Current wallet account
+        </Text>
+      </Box>
+      <Text color='gray.1'
+        variant='b1'>
+        {selectedAccount && toShortAddress(selectedAccount, { size: 33 })}
```

Comment:
I think this needs to be the account we're importing, rather than the selected account. I'm enquiring about that on figma.

Reply:
@wahidrahim let's display the account being imported rather than the selected one. The latter doesn't make any sense in this context.

As for the title of that input, it could read "**Account address**".